docs/lightning/02-installation/02-guided-setup.md

```diff
@@ -66,11 +66,11 @@ with OAuth settings enabled.
 7.  In the Selected OAuth Scopes section, select the following and add
     them to the Selected OAuth Scopes:
 
-8.  Access and manage your data (api)
+8.  Access the identity URL service (id, profile, email, address, phone)
 
-9.  Access your basic information (id, profile, email, address, phone)
+9.  Manage user data via APIs (api)
 
-10. Select the checkbox for Require Secret for Web Server Flow
+10. Select the checkbox for Require Secret for Web Server Flow, and the checkbox for Require Secret For Refresh Token Flow
```

Comment:
Is the image on line 77 up to date or should you be using `/img/classic/image105.png`? If you're gonna use the new image, can you move it to the shared folder too?